control-plane-operator/controllers/hostedcontrolplane/pki/mcs.go

```diff
@@ -11,8 +11,10 @@ import (
 func ReconcileMachineConfigServerCert(secret, ca *corev1.Secret, ownerRef config.OwnerRef) error {
 	hostNames := []string{
 		"machine-config-server",
+		fmt.Sprintf("machine-config-server.%s", secret.Namespace),
 		fmt.Sprintf("machine-config-server.%s.svc", secret.Namespace),
 		fmt.Sprintf("machine-config-server.%s.svc.cluster.local", secret.Namespace),
+		fmt.Sprintf("*.machine-config-server.%s.svc.cluster.local", secret.Namespace),
```

Comment:
do we really need the others domains?

Reply:
Handling request using "*.machine-config-server.%s.svc.cluster.local", other one kept it just for failsafe But can be removed.

Reply:
we can remove the others that's fine just need the final wildcard one

Reply:
This has been fixed

Reply:
>This has been fixed

I can still see all of them in this diff